README.md

```diff
@@ -48,6 +48,7 @@ Several compilation switches are used:
  - LWM2M_BOOTSTRAP_SERVER_MODE to enable LWM2M Bootstrap Server interfaces.
  - LWM2M_BOOTSTRAP to enable LWM2M Bootstrap support in a LWM2M Client.
  - LWM2M_SUPPORT_JSON to enable JSON payload support (implicit when defining LWM2M_SERVER_MODE)
+ - LWM2M_SUPPORT_DTLS to enable DTLS support (require tinydtls)
```

Comment:
Since this flag is used only in the example client, I would prefer it to not appear here. It would confuse people in thinking the core supports DTLS. And maybe rename it to something more client specific.